java/src/lib.rs

```diff
@@ -176,3 +179,76 @@ pub extern "system" fn Java_glide_ffi_resolvers_ScriptResolver_dropScript<'local
     let hash_str: String = env.get_string(&hash).unwrap().into();
     glide_core::scripts_container::remove_script(&hash_str);
 }
+
+impl From<logger_core::Level> for Level {
```

Comment:
Why we can't reuse `Level` from logger core?

Reply:
As in pass in the logger_core::Level from Java? I don't think that's possible. It's not even `repr(C)`. This conversion is also defined in Python.

Reply:
I got this working without defining one more `Level` enum
`logger_core::Level <=> int <=> Logger.Level`

Reply:
I don't define a new Level enum. This is a newtype wrapper, which is erased at compile time.